docs/10_Overview_and_Usage.md

```diff
@@ -1104,6 +1130,48 @@ The Fellowship Of The Ring,54453,13.99,J.R.R. Tolkien
 Dune,143535,14.99,Frank Herbert
 ```
 
+### Export
+
+To export all the elements of a collection you have to use the `GET /export` method exposed by each endpoint associated with a collection. It opens a data stream in different formats.
+
+#### Headers
+
+The export format can be specified through the `Accept` header, supported formats (and relative headers) are the following ones:
+
+| Format | Accept Header value | Description |
+|--------|---------------------|-------------|
+| `json` | `application/json` | Data is exported in JSON format. |
+| [`ndjson`](https://en.wikipedia.org/wiki/JSON_streaming#Newline-delimited_JSON) | `application/x-ndjson` | Data is exported in JSON, each record is processed individually and separated by a newline (`\n`) delimiter. |
+| `csv` | `text/csv` | Data is exported in CSV format using comma as separator as default (the CSV includes the header with column names) |
+| `xlsx` | `application/vnd.openxmlformats-officedocument.spreadsheetml.sheet` | Data is exported in XLS format (the file includes the header with column names) |
+| `xls` | `application/vnd.ms-excel` | Data is exported in XLS format (the file includes the header with column names) |
+
+
+#### Query parameters
+
+It accepts all the query parameters accepted by the `GET /` method plus the `_exportOpts` used for specify the separator for the `csv` format.
+
+```javascript
+  _exportOpts: {
+    type: 'object',
+    properties: {
+      delimiter: string
+    }
+  }
+```
+
+#### Example
+
+The route to call is the following:
+
+`GET` `https://your-url/<CRUD collection endpoint>/export`
```

Comment:
Is it possible to provide the example as curl (there should be something similar across service doc)? In this manner user can "just" copy the call and test it after launching the service locally.